src/librustdoc/html/static/js/src-script.js

```diff
@@ -100,11 +97,16 @@ window.rustdocToggleSrcSidebar = () => {
 
 // This function is called from "src-files.js", generated in `html/render/write_shared.rs`.
 // eslint-disable-next-line no-unused-vars
-function createSrcSidebar() {
+/**
+ * @param {string} srcIndexStr - strinified json map from crate name to dir structure
+ */
+function createSrcSidebar(srcIndexStr) {
+    console.log(srcIndexStr);
```

Comment:
Leftover from debugging?
```suggestion
```

Reply:
Good catch! We should probably make tidy catch this.